src/psyclone/tests/psyir/transformations/transformations_test.py

```diff
@@ -175,6 +183,45 @@ def test_parallellooptrans_refuse_codeblock():
     assert ("Nodes of type 'CodeBlock' cannot be enclosed "
             "by a OMPParallelLoopTrans transformation" in str(err.value))
 
+
+# Tests for OMPSingleTrans
+def test_ompsingle():
+    ''' Generic tests for the OMPSingleTrans transformation class '''
+    trans = OMPSingleTrans()
+    assert trans.name == "OMPSingleTrans"
+    assert str(trans) == "Insert an OpenMP Single region"
+
+    assert not trans.omp_nowait
+    trans.omp_nowait = True
+    assert trans.omp_nowait
+
+
+def test_ompsingle_invalid_nowait():
+    ''' Tests to check OMPSingle rejects invalid attempts
+        to pass nowait argument '''
+    trans = OMPSingleTrans()
+    with pytest.raises(TypeError) as err:
+        trans.omp_nowait = "string"
+    assert ("Expected nowait to be a bool but got string"
```

Comment:
Please could you add single quotes around the thing that has been supplied (so "'string'" here). Possibly better, maybe change to say what type the thing is (using `type(thing).__name__` for Python 2/3 compatibility).

Reply:
Done - it now says the type supplied instead of the value - i didn't put quotes because of that.